modules/nodes-containers-sysctls-about.adoc

```diff
@@ -44,7 +44,10 @@ version and distributor.
 Sysctls that are not namespaced are called _node-level_ and must be set
 manually by the cluster administrator, either by means of the underlying Linux
 distribution of the nodes, such as by modifying the *_/etc/sysctls.conf_* file,
-or by using a daemon set with privileged containers.
+or by using a daemon set with privileged containers. You can use
+xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[the Node Tuning Operator]
```

Comment:
@jmencak The build might be failing because there is an xref used in this module. Unfortunately, right now due to tooling we aren't able to include xrefs in modules, so you will have to remove this link. The use of an xref in the other file is okay because that is at an assembly level.